servers/src/main/java/tachyon/master/Master.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.master;
+
+import java.io.IOException;
+
+import org.apache.thrift.TProcessor;
+
+import tachyon.master.journal.JournalEntry;
+import tachyon.master.journal.JournalInputStream;
+import tachyon.master.journal.JournalSerializable;
+
+public interface Master extends JournalSerializable {
+  /**
+   * Returns the thrift processor for this master.
+   *
+   * @return the {@link TProcessor} serving this master.
+   */
+  TProcessor getProcessor();
+
+  /**
+   * Returns the service name for this master.
+   *
+   * @return a {@link String} representing this master service name.
+   */
+  String getProcessorName();
```

Comment:
Is there a gap between the naming on the client and master side (`service name` vs `processor name`)?